src/components/Ideas/IdeaForm.js

```diff
@@ -84,6 +87,7 @@ export const IdeaForm = () => {
                     <select value={idea.artistId} id="artistId" name="artistId" className="form-control"
                     onChange={handleControlledInputChange}>
                         <option value="0">Select Tattoo Artist</option>
+                        <option value="1">Undecided</option>
```

Comment:
Also, if you have Undecided as to the value of `artistId`, this will not be selected. The value would need to be "1" in order for this to be selected by default.